From 088c7f808b483abf3567adace81ebe929d04e42e Mon Sep 17 00:00:00 2001
From: Sabri Ünal <yakushabb@gmail.com>
Date: Tue, 4 Jun 2024 22:05:35 +0300
Subject: [PATCH] appdata: Update appdata

- Remove categories from appdata file which already provided via desktop file
- Add missing default screenshot
- Fix updatecontact tag
---
 data/net.danigm.timetrack.appdata.xml.in.in | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/data/net.danigm.timetrack.appdata.xml.in.in b/data/net.danigm.timetrack.appdata.xml.in.in
index 10168cd..8beb821 100644
--- a/data/net.danigm.timetrack.appdata.xml.in.in
+++ b/data/net.danigm.timetrack.appdata.xml.in.in
@@ -11,12 +11,8 @@
     <p>Timetrack is a simple time track application for GNOME</p>
  </description>
 
- <categories>
-   <category>Utility</category>
- </categories>
-
  <screenshots>
-    <screenshot>
+    <screenshot type="default">
       <image>https://gitlab.gnome.org/danigm/timetrack/raw/master/screenshots/timetrack1.png</image>
     </screenshot>
     <screenshot>
@@ -196,7 +192,7 @@
    </releases>
 
  <url type="homepage">https://gitlab.gnome.org/danigm/timetrack</url>
- <updatecontact>danigm@gnome.org</updatecontact>
+ <update_contact>danigm@gnome.org</update_contact>
  <developer_name>Daniel García Moreno</developer_name>
 </component>
 
--
libgit2 1.7.2

